ios/KlarnaOnsiteMessagingView.swift

```diff
@@ -10,82 +10,78 @@ import KlarnaMobileSDK
 
 class KlarnaOnsiteMessagingView : UIView {
     private var osmView: KlarnaOSMView
-    
+
     // It couldnt be changed on the fly, but if u will change it - it will force re-render
     @objc var clientId: String = "" {
         didSet {
             setNeedsDisplay()
         }
     }
-    
+
     @objc var placementKey: String = "" {
         didSet {
             setNeedsDisplay()
         }
     }
-    
+
     @objc var locale: String = "" {
         didSet {
             setNeedsDisplay()
         }
     }
-    
+
     @objc var environment: Int = 0 {
         didSet {
             setNeedsDisplay()
         }
     }
-    
+
     @objc var region: Int = 0 {
         didSet {
             setNeedsDisplay()
         }
     }
-    
+
     @objc var purchaseAmount: NSInteger = NSInteger() {
         didSet {
             setNeedsDisplay()
         }
     }
-    
+
     @objc var onHeightChange: RCTBubblingEventBlock?
-    
+
     @objc var onOSMViewError: RCTBubblingEventBlock?
-    
+
     init() {
         self.osmView = KlarnaOSMView()
-        
+
         super.init(frame: .zero)
     }
-    
+
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-    
+
     override func draw(_ rect: CGRect) {
-        if rect == self.osmView.frame {
-            return
-        }
-        
```

Comment:
Curious if you know why this package was returning from `draw` for the OSM view?

Reply:
Probably to prevent needless re-renders. Problem with that though is that when props change they never propagate to `osmView`.